examples/tutorials/curation/plot_1_automated_curation.py

```diff
@@ -0,0 +1,248 @@
+
+print(model.feature_names_in_)
+
+##############################################################################
+# Hence, to use this model we need to create a ``sorting_analyzer`` with all these metrics computed.
+# We'll do this by generating a recording and sorting, creating a sorting analyzer and computing a
+# bunch of extensions. Follow these links for more info on `recordings <https://spikeinterface.readthedocs.io/en/latest/modules/extractors.html>`_, `sortings <https://spikeinterface.readthedocs.io/en/latest/modules/sorters.html>`_, `sorting analyzers <https://spikeinterface.readthedocs.io/en/latest/tutorials/core/plot_4_sorting_analyzer.html#sphx-glr-tutorials-core-plot-4-sorting-analyzer-py>`_
+# and `extensions <https://spikeinterface.readthedocs.io/en/latest/modules/postprocessing.html>`_.
+
+recording, sorting = si.generate_ground_truth_recording(num_channels=4, seed=4, num_units=10)
+sorting_analyzer = si.create_sorting_analyzer(sorting=sorting, recording=recording)
+sorting_analyzer.compute(['noise_levels','random_spikes','waveforms','templates','spike_locations','spike_amplitudes','correlograms','principal_components','quality_metrics','template_metrics'])
+
+##############################################################################
+# This sorting_analyzer now contains the required quality metrics and template metrics.
+# We can check that this is true by accessing the extension data.
+
+all_metric_names = list(sorting_analyzer.get_extension('quality_metrics').get_data().keys()) + list(sorting_analyzer.get_extension('template_metrics').get_data().keys())
+print(set(all_metric_names) == set(model.feature_names_in_))
+
+##############################################################################
+# Great! We can now use the model to predict labels. You can either pass a HuggingFace repo, or a
+# local folder containing the model file and the ``model_info.json`` file. Here, we'll pass
+# a repo. The function returns a dictionary containing a label and a confidence for each unit
+# contained in the ``sorting_analyzer``.
+
+labels = sc.auto_label_units(
+    sorting_analyzer = sorting_analyzer,
+    repo_id = "SpikeInterface/toy_tetrode_model",
+    trusted = ['numpy.dtype']
+)
+
+labels
+
+
+##############################################################################
+# The model has labelled one unit as bad. Let's look at that one, and the 'good' unit with the highest
+# confidence of being 'good'.
+
+sw.plot_unit_templates(sorting_analyzer, unit_ids=[7,9])
+
+##############################################################################
+# Nice - we see that unit 9 does look a lot more spikey than unit 7. You might think that unit
+# 7 is a real unit. If so, this model isn't good for you.
+#
+# Assess the model performance
+# ----------------------------
+#
+# To assess the performance of the model relative to human labels, we can load or generate some
```

Comment:
maybe 'labels assigned by a human curator'? (just the first time the concept is mentioned)